makefile

```diff
@@ -0,0 +1,37 @@
+IMAGE := cloud-platform-slack-prbot
+VERSION := 1.0
+ORG := ministryofjustice
+TAGGED := $(ORG)/$(IMAGE):$(VERSION)
+
+.built-docker-image: Dockerfile Gemfile config.ru web.rb lib/pr.rb $(wildcard lib/pr/*.rb)
+	docker build -t $(IMAGE) .
+	docker tag $(IMAGE) $(TAGGED)
+	touch .built-docker-image
```

Comment:
It'd be nice to have a corresponding `make clean` that removes the built docker image and the file you touch here.

Reply:
Good idea.